dockstore/dockstore-ui2
# Redirect to registered workflow after quick register

**Description**
This PR redirects the user to their most recently registered workflow after they complete quick registration.

Note that I didn't encounter the sizing issue mentioned in the ticket locally or on QA. It must've been taken care of at some point during the redesign.

**Review Instructions**
On QA,
- Go to My Workflows and click register a workflow
- Select the Quickly register remote workflows option
- Register one of the workflows in your repository by toggling one of the options
- Click Finish
- Verify that you're redirected to the workflow you just registered

**Issue**
dockstore/dockstore#4885
[DOCK-2161](https://ucsc-cgl.atlassian.net/browse/DOCK-2161)

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!



[DOCK-2161]: https://ucsc-cgl.atlassian.net/browse/DOCK-2161?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
